examples/tunables_limit_memory.rs

```diff
@@ -152,7 +154,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     let tunables = LimitingTunables::new(base, Pages(24));
 
     // Create a store, that holds the engine and our custom tunables
-    let store = Store::new_with_tunables(&engine, tunables);
```

Comment:
Rather than adding an argument to the already existing new_with, make a new function called `new_with_tunables_and_path` or similar